diff --git a/evals/terminal-bench-2.yaml b/evals/terminal-bench-2.yaml
index b0e48d2..d42a6fd 100644
--- a/evals/terminal-bench-2.yaml
+++ b/evals/terminal-bench-2.yaml
@@ -44,6 +44,7 @@ datasets:
       - terminal-bench/pypi-server
       - terminal-bench/schemelike-metacircular-eval
       - terminal-bench/distribution-search
+      - terminal-bench/largest-eigenval
 
 environment:
   import_path: harbor_adapter.environment:FastDockerEnvironment
diff --git a/harbor_adapter/verifier.py b/harbor_adapter/verifier.py
index cf45d07..cbe6461 100644
--- a/harbor_adapter/verifier.py
+++ b/harbor_adapter/verifier.py
@@ -59,6 +59,8 @@ class PytestVerifier(Verifier):
             "pip() { "
             'case "$*" in '
             '"install pytest==8.4.1 pytest-json-ctrf==0.3.5"|'
+            '"install pytest==8.4.1 pytest-json-ctrf==0.3.5 '
+            '--break-system-packages"|'
             '"install pytest==8.4.1 requests==2.32.5 '
             'pytest-json-ctrf==0.3.5"|'
             '"install pytest==8.4.2 requests==2.32.5 psutil==7.0.0 '
