diff --git a/evals/terminal-bench-2.yaml b/evals/terminal-bench-2.yaml
index 166cc03..0ca1058 100644
--- a/evals/terminal-bench-2.yaml
+++ b/evals/terminal-bench-2.yaml
@@ -47,6 +47,7 @@ datasets:
       - terminal-bench/largest-eigenval
       - terminal-bench/constraints-scheduling
       - terminal-bench/write-compressor
+      - terminal-bench/tune-mjcf
 
 environment:
   import_path: harbor_adapter.environment:FastDockerEnvironment
diff --git a/harbor_adapter/verifier.py b/harbor_adapter/verifier.py
index cbe6461..fcc2ce4 100644
--- a/harbor_adapter/verifier.py
+++ b/harbor_adapter/verifier.py
@@ -86,6 +86,8 @@ class PytestVerifier(Verifier):
             '-w pytest-json-ctrf==0.3.5 pytest "*|'
             '"-p 3.13 -w pytest==8.4.1 -w numpy==2.3.0 '
             '-w pytest-json-ctrf==0.3.5 pytest "*|'
+            '"-p 3.13 -w pytest==8.4.1 -w mujoco==3.3.5 '
+            '-w pytest-json-ctrf==0.3.5 pytest "*|'
             '"-p 3.13 -w pytest==8.4.1 -w rdflib==7.1.4 '
             '-w pytest-json-ctrf==0.3.5 pytest "*|'
             '"-p 3.13 -w pytest==8.4.1 -w requests==2.32.4 '
