diff --git a/evals/terminal-bench-2.yaml b/evals/terminal-bench-2.yaml
index e1b116f..c5a35f1 100644
--- a/evals/terminal-bench-2.yaml
+++ b/evals/terminal-bench-2.yaml
@@ -41,6 +41,7 @@ datasets:
       - terminal-bench/merge-diff-arc-agi-task
       - terminal-bench/winning-avg-corewars
       - terminal-bench/sparql-university
+      - terminal-bench/pypi-server
 
 environment:
   import_path: harbor_adapter.environment:FastDockerEnvironment
diff --git a/harbor_adapter/verifier.py b/harbor_adapter/verifier.py
index 28dc25a..58095eb 100644
--- a/harbor_adapter/verifier.py
+++ b/harbor_adapter/verifier.py
@@ -80,6 +80,8 @@ class PytestVerifier(Verifier):
             "uvx() { "
             'case "$*" in '
             '"-p 3.13 -w pytest==8.4.1 -w pytest-json-ctrf==0.3.5 pytest "*|'
+            '"-p 3.13 -w pytest==8.4.1 -w pip==25.2 '
+            '-w pytest-json-ctrf==0.3.5 pytest "*|'
             '"-p 3.13 -w pytest==8.4.1 -w rdflib==7.1.4 '
             '-w pytest-json-ctrf==0.3.5 pytest "*|'
             '"-p 3.13 -w pytest==8.4.1 -w requests==2.32.4 '
